tests/financial/test_us_new_york.py

```
@@ -0,0 +1,267 @@
+from datetime import date
+from unittest import TestCase
+
+from holidays.calendars.gregorian import JAN, FEB, MAY, JUN, JUL, SEP, OCT, NOV, DEC, _timedelta
+from holidays.constants import HALF_DAY, PUBLIC
+from holidays.financial.us_new_york import USNewYork, USNY, USNewYorkFinancial
+from tests.common import CommonFinancialTests
+
+
+class TestUSNewYork(CommonFinancialTests, TestCase):
+    @classmethod
+    def setUpClass(cls):
```

Ruff (0.14.1)

Missing return type annotation for classmethod `setUpClass`

Add return type annotation: `None`

(ANN206)